project/frontend/src/containers/Dashboard/Dashboard.js

```diff
@@ -56,56 +57,43 @@ const Dashboard = (props) => {
     },
   ];
 
-  const placeholderTrends = [
-    "Kellyanne Conway",
-    "Filibuster",
-    "Fultondale al",
-    "Ben Askren",
-    "Stock-market",
-    "Etsy stock",
-    "Elliot Page",
-    "Susan Rice",
-    "Raya and the Last Dragon",
-    "Starlink",
-  ];
+  const toDayHourMinute = (totalTime) => {
```

Comment:
Is there a library do some of this date/time formatting?

Reply:
There are no obvious ones that do what I want it to do, and I figured that less dependencies is better with something that's so easily done.